src/EnergyPlus/Psychrometrics.cc

```diff
@@ -1439,6 +1444,26 @@ namespace Psychrometrics {
 
         return Temp;
     }
+    Real64 CSplineint(int const n,     // sample data size
+                      const Real64 &x) // given value of x
```

Comment:
There is no reason to pass scalars by `const &`. If this is only an input, it should be passed by value. The only parameters that should be passed by reference are output parameters (passing by reference is the only) and objects (passing by value will trigger the copy constructor which 99% of the time you don't want to do).

Reply:
@amirroth Many thanks for your comments and the details! I will drop the `const &`.